packages/react-utilities/src/utils/on.ts

```diff
@@ -0,0 +1,9 @@
+export const on = <T>(
+  element: Element | Window | Document,
+  eventName: string,
+  callback: (ev: T) => void,
+  options?: boolean,
```

Comment:
If this is just a boolean it should be called `useCapture`. (There are more options available, but the param could be renamed later without a break if we want to add support for them.) https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener